Library/Homebrew/cli/parser.rb

```diff
@@ -590,12 +614,13 @@ def check_conflicts
           end
 
           select_cli_arg = violations.count - env_var_options.count == 1
-          raise OptionConflictError, violations.map(&method(:name_to_option)) unless select_cli_arg
+          raise OptionConflictError, violations.map { name_to_option(_1) } unless select_cli_arg
```

Comment:
sorbet requires these to be removed under `strict` typing, to validate method arity